src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -779,5 +776,52 @@ static void WritePropertyAccessorModifiers(CodeWriter writer, MethodSignatureMod
                     .AppendRawIf("private ", modifiers.HasFlag(MethodSignatureModifiers.Private));
             }
         }
+
+        public static void WriteTypeArguments(this CodeWriter writer, IEnumerable<CSharpType>? typeArguments)
+        {
+            if (typeArguments is null)
+            {
+                return;
+            }
+
+            writer.AppendRaw("<");
```

Comment:
do we need a TODO to normalize this with the other write arguments that handles the docs case where instead of `<` we use `{`?

Reply:
Done